index.html

```diff
@@ -234,10 +235,26 @@ <h6 class="h6">API Desing</h6>
                     <h6 class="h-last h6">Web Development</h6>
                 </div>
         </section>
-    </main>
-
-    
-    <footer>
-    </footer>
+        <!-- Added form on the last page-->
+        <section class="section-11">
+                <h5 class="h5-2">I'm always interested in hearing 
+                about new projects, so if you'd like to chat 
+                please get in touch.</h5>
+                <form action="https://formspree.io/f/xayzvjyw" method="post" class="form">
+                    <input type="text" name="full_name" id="name" placeholder="Full name" required
+                    minlength="30">
```

Comment:
- [ ] You have used `minlength="30"` here. But in the requirements, I can see it says, "Name field accepts `up to 30 chars.`" `Up to` means `Maximum`. So kindly replace with `maxlength="30"` to fix this issue.
- ![image](https://user-images.githubusercontent.com/60083437/235795755-1651933c-ee55-48d3-a4e0-cab62df94c06.png)